aries-site/src/layouts/main/Header.js

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 import { AnchorGroup, Nav } from 'aries-core';
 
-export const Header = ({ showLinks }) => (
-  <Nav title="Design System">
+export const Header = ({ background, showLinks }) => (
+  <Nav title="Design System" background={background}>
```

Comment:
I'm OK with that change, I'm just thinking that if we'll have `...rest` as a prop we won't need the deconstruction of the `background` prop.